src/offset/fixed.rs

```diff
@@ -24,7 +24,7 @@ use crate::Timelike;
 #[derive(PartialEq, Eq, Hash, Copy, Clone)]
 #[cfg_attr(feature = "rkyv", derive(Archive, Deserialize, Serialize))]
 pub struct FixedOffset {
-    local_minus_utc: i32,
+    pub(crate) local_minus_utc: i32,
```

Comment:
This commit should be split in a number of smaller ones:

* Use the `offset()` method instead of directly accessing the `ut_offset` field
* Change `LocalResult` return values from `LocalTimeType` to offset only

Renaming any identifier should be a commit by itself.